modules/integration/src/test/scala/zio/elasticsearch/HttpExecutorSpec.scala

```diff
@@ -1477,7 +1551,7 @@ object HttpExecutorSpec extends IntegrationSpec {
             Executor.execute(ElasticRequest.createIndex(firstSearchIndex)),
             Executor.execute(ElasticRequest.deleteIndex(firstSearchIndex)).orDie
           ),
-          test("search for a document using script query") {
+          test("search for a document using should without satisfying minimumShouldMatch condition") {
```

Comment:
Can you omit: `without satisfying minimumShouldMatch condition`? If that makes sense.